### PR TITLE
[5.0] Upmerge

### PR DESCRIPTION
**Pull request by laoneo**

### CODE REVIEW

**Comment by richard67** [comment]
System tests are failing.

File: tests/System/integration/administrator/components/com_users/Groups.cy.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    cy.contains('User Group Details');
+  });
+
+  it('can delete the test group', () => {
+    cy.db_createUserGroup({ title: 'Test group' }).then(() => {
+      cy.searchForItem('Test group');
+      cy.checkAllResults();
+      cy.clickToolbarButton('Delete');
+      cy.on('window:confirm', () => true);
```
Comment (heelc29) [inline tests/System/integration/administrator/components/com_users/Groups.cy.js]:
```suggestion
      cy.clickDialogConfirm(true);
```

File: tests/System/integration/administrator/components/com_users/Levels.cy.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    cy.contains('Level Details');
+  });
+
+  it('can delete the test level', () => {
+    cy.db_createUserLevel({ title: 'Test level' }).then(() => {
+      cy.searchForItem('Test level');
+      cy.checkAllResults();
+      cy.clickToolbarButton('Delete');
+      cy.on('window:confirm', () => true);
```
Comment (heelc29) [inline tests/System/integration/administrator/components/com_users/Levels.cy.js]:
```suggestion
      cy.clickDialogConfirm(true);
```

**Review by heelc29 — COMMENTED** [review]
Changes for https://github.com/joomla/joomla-cms/pull/41910
See also https://github.com/joomla/joomla-cms/pull/41434

**Comment by richard67** [comment]
It seems 3 tests failed:
1) can invalidate privacy consent
2) can invalidate all privacy consents
3) can displays correct number of consents

**Pull request merged**